apps/disease_control/templates/includes/disease_control_card.html

```diff
@@ -1,10 +1,48 @@
 {% load wagtailcore_tags %}
 
-<div class="disease-control-card-hip columns">
-  <div class="column is-4">
-    <a class="card-link-hip is-size-5" href="{{ page.get_url }}"><strong>{{ page.title }}</strong></a>
-  </div>
-  <div class="page-description-hip column">
-    {{ page.description|richtext }}
+<div class="disease-control-card-hip py-4">
+  <div class="columns">
+    <div class="column is-4">
+      <a class="card-link-hip is-size-5 is-block" href="{{ page.get_url }}"><strong>{{ page.title }}</strong></a>
+      {% if page.diseasepage %}
+        <p class="emergent-dates-hip pt-2">
+          {% if page.specific.emergent_begin_date %}
+            {{ page.specific.emergent_begin_date }} - 
+          {% endif %}
+          {% if page.specific.emergent_end_date %}
+             {{ page.specific.emergent_end_date }}
+          {% elif page.specific.emergent_begin_date and not page.specific.emergent_end_date %}
+            Present
+          {% endif %}
+        </p>
+        {% if page.specific.is_emergent and not page.specific.description %}
+          <a class="button is-rounded is-small mt-2 emergent-btn-hip" href="{{ page.get_url }}">Emergent</a>
+        {% endif %}
+      {% endif %}
+    </div>
+    {% if page.diseasepage %}
+      <div class="page-description-hip column">
+        {{ page.specific.description|richtext }}
+        {% if not page.specific.description %}
+          <span class="is-block">Updated {{ page.latest_revision_created_at|date:"M d, Y" }}</span>
+        {% endif %}
+      </div>
+    {% else %}
+      <div class="page-description-hip column">
+        {{ page.description|richtext }}
+      </div>
+    {% endif %}
   </div>
+  {% if page.diseasepage and page.specific.description %}
+    <div class="columns">
+      {% if page.specific.is_emergent %}
```

Comment:
This `if` conditional should go inside the next `div`, otherwise this div gets hidden on non-emergent pages, so the `Updated` string ends up in the left column:
![image](https://user-images.githubusercontent.com/4413/109823746-370a4500-7c06-11eb-9b3d-7799a51da119.png)

Reply:
Thanks @vkurup. Pushed the change.